mailpoet/lib/Config/PrivacyPolicy.php

```diff
@@ -42,28 +42,44 @@ public function getPrivacyPolicyContent() {
+          sprintf(__('Cookie name: %s', 'mailpoet'), 'mailpoet_page_view' ) .
+          '<br>' .
+          // translators: %s is the number of days.
+          sprintf(__('Cookie expiry: %s days.', 'mailpoet'), WPFunctions::get()->numberFormatI18n(3650) ) .
+          '<br>' .
+          __('Cookie description: The purpose of this cookie is to track the last time a subscriber viewed any page on the site.', 'mailpoet') .
         '</p> ' .
         '<p>' .
-        // translators: %s is the name of the cookie.
-        sprintf(__('Cookie name: %s', 'mailpoet'), 'mailpoet_abandoned_cart_tracking' ) .
-        '<br>' .
-        // translators: %s is the number of days.
-        sprintf(__('Cookie expiry: %s days.', 'mailpoet'), WPFunctions::get()->numberFormatI18n(3650) ) .
-        '<br>' .
-        __('Cookie description: The purpose of this cookie is to track a user that has abandoned their cart in your WooCommerce store to then be able to send them an abandoned cart newsletter from MailPoet.', 'mailpoet') .
-        '<br>' .
-        '<br>' .
-        __('Note: User must be opted-in and a confirmed subscriber.', 'mailpoet') .
+          // translators: %s is the name of the cookie.
+          sprintf(__('Cookie name: %s', 'mailpoet'), 'mailpoet_revenue_tracking' ) .
+          '<br>' .
+          // translators: %s is the number of days.
+          sprintf(__('Cookie expiry: %s days.', 'mailpoet'), WPFunctions::get()->numberFormatI18n(14) ) .
+          '<br>' .
+          __('Cookie description: The purpose of this cookie is to track which newsletter sent from your website has acquired a click-through and a subsequent purchase in your WooCommerce store.', 'mailpoet') .
+        '</p> ' .
+        '<p>' .
+          // translators: %s is the name of the cookie.
+          sprintf(__('Cookie name: %s', 'mailpoet'), 'mailpoet_subscriber' ) .
+          '<br>' .
+          // translators: %s is the number of days.
+          sprintf(__('Cookie expiry: %s days.', 'mailpoet'), WPFunctions::get()->numberFormatI18n(3650) ) .
+          '<br>' .
+          __('Cookie description: The purpose of this cookie is to track subscriber engagement. It is used when the user logs in, signs up in a form, confirms subscription to a newsletter, or places an order through WooCommerce.', 'mailpoet') .
+          '<br>' .
+          __('Note: User must be opted-in and a confirmed subscriber.', 'mailpoet') .
+        '</p>' .
+        '<p>' .
+          // translators: %s is the name of the cookie.
+          sprintf(__('Cookie name: %s', 'mailpoet'), 'popup_form_dismissed_{$formId}' ) .
+          '<br>' .
+          __('Cookie expiry: the expiration date varies and can be set per form.', 'mailpoet') .
+          '<br>' .
+          __('Cookie description: This cookie is used to track whether the user has already subscribed to a specific form. It is applicable for popup, slide-in, or fixed bar forms.', 'mailpoet') .
```

Comment:
```suggestion
          __('Cookie description: This cookie is used to track if a user has previously dismissed a specific form, preventing the re-display of the form until the cookie's expiration date. It is applicable for popup, slide-in, or fixed bar forms.', 'mailpoet') .
```
Something like that?

Reply:
Thanks, Brezo. I added a new commit implementing your suggestion.